### ISSUE TITLE
There was an error: Authentication timed out

### ISSUE THREAD

**Issue by camstuart** [feathersjs/feathers-chat#67]
A fresh clone and `npm install` contains an error when attempting to authenticate:

Steps to reproduce:

1. clone a fresh copy of the repo
2. `cd feathers-chat && yarn`
3. `yarn start`
4. Visit `http://localhost:3030/jquery` or `http://localhost:3030/vanilla`
5. Create a user account
6. Attempt to sign in with newly created credntials

Error is (server console)

```
info: after: users - Method: find
error: Unhandled Rejection at: Promise  Promise {
  <rejected> TypeError: done is not a function
    at /home/cameron/sde/cam/feathers-chat/node_modules/feathers-authentication-local/lib/verifier.js:136:38 } TypeError: done is not a function
    at /home/cameron/sde/cam/feathers-chat/node_modules/feathers-authentication-local/lib/verifier.js:136:38
```

**Comment by daffl** [comment]
Thank you! It should be fixed if you pull latest. Apparently it used an old version of `feathers-authentication` which I fixed it in #68

**Comment by camstuart** [comment]
Thanks @daffl
